server/conf/i18n/messages

```diff
@@ -195,8 +211,12 @@ label.searchByDob=Search by Date of Birth
 label.searchByName=Search by name(s)
 # The label above the input field for the year of the client birth date.
 label.year=Year
+# Label showing the user when there is no name
+label.unnamedUser=Unnamed user
 
 link.edit=Edit
+# Link to select a new client in the toast message when viewing an application
```

Comment:
suggestion (blocking):  Since this is a banner and not a toast message, I think it would be clearer to say "Link to change the client that the trusted intermediary is applying for."

Reply:
suggestion applied. thank you!